crates/dojo-lang/src/semantics/test_data/get

```diff
@@ -0,0 +1,408 @@
+                        __Health_model_span__,
+                    ),
+                    expr: FunctionCall(
+                        ExprFunctionCall {
+                            function: ?12::span,
+                            args: [
+                                Value(
+                                    Snapshot(
+                                        ExprSnapshot {
+                                            inner: Var(
+                                                LocalVarId(test::__Health_model__),
+                                            ),
+                                            ty: @core::array::Array::<core::felt252>,
+                                        },
+                                    ),
+                                ),
+                            ],
+                            ty: core::array::Span::<?10>,
+                        },
+                    ),
+                },
+            ),
+            Let(
+                StatementLet {
+                    pattern: Variable(
+                        __Health,
+                    ),
+                    expr: FunctionCall(
+                        ExprFunctionCall {
+                            function: ?13::expect,
+                            args: [
+                                Value(
+                                    FunctionCall(
+                                        ExprFunctionCall {
+                                            function: ?14::deserialize,
+                                            args: [
+                                                Reference(
+                                                    LocalVarId(test::__Health_model_span__),
+                                                ),
+                                            ],
+                                            ty: core::option::Option::<<missing>>,
+                                        },
+                                    ),
+                                ),
+                                Value(
+                                    Literal(
+                                        ExprLiteral {
+                                            value: 7624190543980043505746182176517533098884860453715398488377751337573,
+                                            ty: ?12,
+                                        },
+                                    ),
+                                ),
+                            ],
+                            ty: ?11,
+                        },
+                    ),
+                },
+            ),
+        ],
+        tail: Some(
+            Var(
+                LocalVarId(test::__Health),
+            ),
+        ),
+        ty: <missing>,
+    },
+)
+
+//! > semantic_diagnostics
+error: Identifier not found.
+ --> get_inline_macro:2:96
+                let mut __get_macro_keys__ = array::ArrayTrait::new();serde::Serde::serialize(@key, ref __get_macro_keys__);
```

Comment:
Can we improve formatting here? i.e. new line. I guess it's not possible to define key / suppress this? It adds noise to the test

Reply:
It's just showing the whole line where stuff happens. I think most straightforward is to add some line breaks in macro expansion. Which is what I've done in the latest commit 😉